dashboards/google-compute-disk/metadata.yaml

```
@@ -0,0 +1,10 @@
+sample-dashboards:
+  -
+    category: Compute Disk
+    id: compute-disk
+    display_name: Disks
+    description: |-
+    This dashboard has charts displaying an inventory of GCE Disks, as well as Volume throughput and IOPS.
```

I think this needs an additional indent

```suggestion
      This dashboard has charts displaying an inventory of GCE Disks, Volume throughput, and IOPS.
```